app/views/ShareListView/Header/index.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
+// @ts-ignore
+// eslint-disable-next-line import/extensions,import/no-unresolved
```

Comment:
No need. There are two ways to solve this: You can either omit one platform (it'll still detect the other one, e.g. if you omit iOS it'll keep working fine, since Android is still specified and it will detect if is Android or not) or you can create a type definition file (e.g. `Header.d.ts`) so it can detect it.

Reply:
I could fix this with this comment: https://stackoverflow.com/a/59972215 when he talks the `Update`

Reply:
```You can either omit one platform (it'll still detect the other one, e.g. if you omit iOS it'll keep working fine, since Android is still specified and it will detect if is Android or not)```

That was what I was referring to :)